### ISSUE TITLE
postgresql_server --check

### ISSUE THREAD

**Issue by thiagotalma** [debops/debops#584]
Hello @drybjed ,

Is it too complicated to make this command work without error?
`debops service/postgresql_server --check --diff -t role::postgresql_server:config`

Currently I get this error:

```
TASK [debops.postgresql_server : Configure PostgreSQL clusters] **************************************************************************************
failed: [db01] (item={'name': 'main', 'port': '5432'}) => changed=false
  item:
    name: main
    port: '5432'
  msg: 'AnsibleUndefinedVariable: ''dict object'' has no attribute ''stdout'''
```

**Comment by drybjed** [comment]
It seems that some task or tasks need to have the `role::postgresql_server:config` tag added so that they are executed in this context, probably the task that checks the PostgreSQL version. I'll check it out soon, or you could try adding it yourself if it works for you. Thanks for the bug report. :-)

**Comment by thiagotalma** [comment]
Unfortunately I have no knowledge to resolve this error.

The role postgresql also results in an error when trying to use --check.

Personally, I do not have the guts to run these 2 roles in production without first checking what will be changed.

**Comment by thiagotalma** [comment]
Hello @drybjed !
Any news here?

**Comment by drybjed** [comment]
Done. I'll backport the fix to the `stable-1.0` branch as well.

**Comment by thiagotalma** [comment]
Thank you! Thank you!!!

If I had known it was so simple, I would have sent the PR myself. 😄
